fill/proto/interactive_filler_test.go

```diff
@@ -1,16 +1,86 @@
-package proto_test
+package proto
```

Comment:
I dropped the `_test` because I needed access to the internal function I wanted to test.